### PR TITLE
Forum

### PR DESCRIPTION
**Pull request by ovari**
Please do not forget to update the mmex.pot file and write information about the fixed bug in the prerelease [page](https://github.com/moneymanagerex/moneymanagerex/releases).

* Fix `&S` accelerator collision with `&Show this dialog box at startup`
* Singular `Forum`
* Homogenize with Menu bar, Help, Community, Forum
* Convert `F&orum` to `&Forum`
* Fix `&F` collision with `&Facebook` to `Face&book`
* Homogenize `Website` tooltip to `Visit MMEX website for the latest news and updates` to be consistent with text shown in status bar when hovering on `Website` option at: Menu bar, Help, Community, Website

- - -
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/6662)

### CODE REVIEW

File: src/appstartdialog.cpp [hunk 1]
```diff
@@ -118,13 +118,12 @@ void mmAppStartDialog::CreateControls()
     itemBoxSizer5->Add(itemButton9, 0, wxGROW | wxALL, 5);
 
     wxButton* itemButton10 = new wxButton(this, wxID_INDEX, _("&Website"));
-    const wxString s = wxString::Format(_("Visit the %s website")
-        , mmex::getProgramName());
+    const wxString s = wxString::Format(_("Visit the MMEX website"));
```
Comment (whalley) [inline src/appstartdialog.cpp:121]: Why this change?

Comment (ovari) [reply src/appstartdialog.cpp]: Thanks for checking. Updated to `Visit MMEX website for the latest news and updates` to be consistent with text shown in status bar when hovering on `Website` option at:
Menu bar, Help, Community, Website
